minitwit_admin/src/App.vue

```diff
@@ -40,39 +41,36 @@ export default {
     initStore();
     const { getColors } = useColors();
     const colors = computed(() => getColors());
-
     const loggedInUser = getLoggedInUser()
     const loggedUser = computed(() => (loggedInUser.value != 0 ? true : false));
-    const loggedOutUser = computed(() =>
-      loggedInUser.value == 0 ? true : false
-    );
+    const loggedOutUser = computed(() => loggedInUser.value == 0 ? true : false);
```

Comment:
Maybe this could be removed to only use the loggedUser computed variable. Maybe isLoggedInUser name would be better

Reply:
tbh I spent some time trying fore the list of sidebarItems to be reactive to the loggedUser changing and somehow I failed, this is the only (and ugly af) way I made it work, if u have any suggestions how to improve it I will be glad